src/model/db/FileUpload.model.ts

```diff
@@ -96,6 +96,16 @@ export class FileUploadModel extends AbstractModel {
         return FileUtils.isFileExpired(this);
     }
 
+    public get fileProtectionLevel(): string {
+        if (this.encrypted) {
+            return "Encrypted";
+        }
+        if (this.settings?.password != null) {
```

Comment:
can just be ` if (this.settings?.password) {`

Reply:
Implemented